README.md

```diff
@@ -1,2 +1,127 @@
+**NOTE:** this is a draft README for not fully implemented yet feature.
+
```

Comment:
Also, fix all markdownlint complaints for this file.
See here -- https://codeclimate.com/github/raccoongang/edx-calendar-tab/pull/1